server.js

```diff
@@ -63,11 +63,41 @@ async function handleEvent(event) {
 
     return client.replyMessage(event.replyToken, {
       type: "text",
-      text: `${event.message.text} さんですね。次は学類を教えてください。`,
+      text: `${answer["name"]} さんですね。次は学類を教えてください。`,
     });
   }
 
   // こんな感じで順番に聞いてく
+
+  // 名前が記録され学類が入力されていないなら、入力されたメッセージは学類
+  if (answer["name"] != null && answer["faculties"] == null) {
+    // 学類を記録
+    answer["faculties"] = event.message.text;
+
+    return client.replyMessage(event.replyToken, {
+      type: "template",
+      template: {
+        type: "bottons",
+        text: `${answer["name"]}さん、${answer["faculties"]} 所属で登録してよろしいですか？`,
+        actions: [
+          {
+            type: "postback",
+            label: "はい",
+            data: ``,
+            displayText: "送信完了しました！入ってくださりありがとうございます！",
+          },
+          {
+            type: "postback",
+            label: "いいえ",
+            data: ``,
+            displayText: "登録が中止されました。",
+          }
+        ]
+      }
+    });
```

Comment:
```suggestion
        ]
      }
    })
    .catch((e) => {
      console.log("Error!");
      console.log(e.originalError.response.data);
    });
```
こんな感じでエラー出力処理を書いてあげると、送ったデータが変なときに↓みたいなエラーがでてわかりやすいです！
```
{
  message: 'The request body has 1 error(s)',
  details: [ { message: 'May not be empty', property: 'messages[0].altText' } ]
}
```